VSRAD.Package/ProjectSystem/Macros/MacroEvaluator.cs

```diff
@@ -161,7 +162,11 @@ private async Task<Result<string>> GetMacroValueAsync(string name, List<string>
                 case RadMacros.GroupSize: value = _debuggerOptions.GroupSize.ToString(); break;
                 case RadMacros.BreakLines:
                     if (_transientValues.BreakLines.TryGetResult(out var breakLines, out var error))
-                        value = string.Join(RadMacros.BreakLineSeparator, breakLines);
+                    {
+                        bool stopOnHit = _debuggerOptions.ResumeMode;
```

Comment:
I'd rename `DebuggerOptions.ResumeMode` to `StopOnHit` because the current name is confusing, at first I assumed `ResumeMode=true` makes breakpoints resumable, but it seems to do the opposite.

Reply:
Thanks, renamed it.